packages/react/src/components/Button/Button.js

```diff
@@ -262,6 +262,14 @@ Button.propTypes = {
    */
   disabled: PropTypes.bool,
 
+  /**
+   * Deprecated in v11. Use 'onClick' instead.
+   */
+  handleClick: deprecate(
```

Comment:
Is `handleClick` an existing prop for `Button` I couldn't find it in the original source file and it seems like we're adding it in here.

Reply:
You're right. I think I mistakenly added it because I saw those event handlers. I'll remove it.